index.html

```diff
@@ -4,14 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
-    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Montserrat&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style/style.css" type="text/css">
 </head>
 <body>
-    <header>
-        
-    </header>
+  <nav class="menu navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#"></a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav mx-auto text-center">
+        <li class="nav-item">
+          <a class="menu__link menu__link--color nav-link" href="#about">O mnie <span class="sr-only">(current)</span></a>
+        </li>
+        <span class="vertical d-none d-lg-block">|</span>
+        <li class="nav-item">
+          <a class="menu__link menu__link--color nav-link" href="#procjects">Projekty</a>
+        </li>
+        <span class="vertical d-none d-lg-block">|</span>
+        <li class="nav-item active">
+          <a class="menu__link menu__link--color nav-link" href="#skills">Umiejętności</a>
+        </li>
+        <span class="vertical d-none d-lg-block">|</span>
+        <li class="nav-item">
+          <a class="menu__link menu__link--color nav-link" href="#contact" tabindex="-1" aria-disabled="true">Kontakt</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <header class="header">
+      <div class="section-welcome mt-md-5">
+        <div class="container">
+          <div class="row">
+            <div class="col-12">
+              <h1 class="headline-welcome text-white text-center"><span class="text-uppercase distinguish">Damian</span>-Web Developer.</h1>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <div  class="arrow">
+              <div class="mt-5 arrow-icon"><img src="images/icons/down-arrow.svg" alt=""></div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- TODO: dodac linki do własnych stron-->
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-md-6">
+              <div class="social-media d-flex justify-content-center justify-content-md-start">
+                  <a href="https://github.com/" target="blank">
```

Comment:
Hey, tutaj musisz dodac atrybut do linkow target="_blank" w prawidlowy sposob:) aby otwieraly sie w nowej zakladce przegladarki. Musisz tez dodac kolejny atrybut do linku rel="noopener noreferrer" aby zamknac dostep do strony za pomoca window.opener (jest to vulnerability, ktora jest bardzo czesto pomijana przez developerow).

prawidlowo wygladajacy link:

                  <a href="https://github.com/" target="blank" rel="noopener noreferrer">